packages/sitecore-jss-cli/src/create/index.ts

```diff
@@ -105,4 +105,40 @@ export function applyNameToProject(
 
       fs.writeFileSync(finalConfigFileName, configXml);
     });
+
+  replacePrefix(projectFolder, name, replaceName);
+}
+
+/**
+ * Returns a string formatted to PascalCase
+ * my-next-sitecore-app becomes MyNextSitecoreApp
+ * @param {string} name
+ */
+export function getPascalCaseName(name: string): string {
+  const temp: string[] = name.split('-');
+  name = temp.map((item: string) => (item = item.charAt(0).toUpperCase() + item.slice(1))).join('');
+  return name;
+}
+
+/**
+ * Called during jss create, this function replaces the sample's prefix with the app's name on Sitecore templates
+ * @param {string} projectFolder Project folder
+ * @param {string} name Name value to replace
+ * @param {string} prefix Prefix of the sample app's template - should match Jss[RAV|Next]Web
+ */
+export function replacePrefix(projectFolder: string, name: string, prefix: string) {
+  console.log(chalk.cyan(`Replacing template prefix with ${getPascalCaseName(name)}`));
```

Comment:
Add '...' for consistency with other logs
```suggestion
  console.log(chalk.cyan(`Replacing template prefix with ${getPascalCaseName(name)}...`));
```

Reply:
> 
> 
> This is working brilliantly. Not sure if you had this planned for "phase 2" (apply to RAV), but we discussed adding a flag to `jss create` to make the prefix opt-in (for backwards compatibility). I think this is still valuable, and I don't think it'd be that much extra work?

Yeah I'll try to add that functionality here. Shouldn't be too bad!

Reply:
https://github.com/Sitecore/jss/pull/800/commits/8bd96971e4647ba33069cbfe5df907c30d916890 should do the trick.